code/modules/mob/new_player/new_player.dm

```diff
@@ -586,11 +586,7 @@
 		client.prefs.language = "None"
 
 /mob/new_player/proc/ViewManifest()
-	var/dat = "<html><body>"
-	dat += "<h4>Crew Manifest</h4>"
-	dat += GLOB.data_core.get_manifest(OOC = 1)
```

Comment:
Also if `get_manifest()` is no longer used, may as well nuke it.

Reply:
Indeed, only `get_manifest_json()` is used for the TGUI stuff, the `get_manifest()` was for the HTML only manifest.